SplitIt/app/src/main/java/project/cs495/splitit/MainActivity.java

```diff
@@ -290,7 +291,9 @@ private String parceScanResults(ScanResults brScanResults, String groupId) {
             item.addReceiptId(receiptId);
             item.commitToDB(database);
             receipt.addItem(item.getItemId());
+            subTotal += item.getPrice();
         }
+        if (receipt.getTax() == 0) receipt.setTax((receipt.getPrice() / subTotal) - 1);
```

Comment:
It shouldn't be necessary to get the rate. We can use the dollar amount of tax (receipt.getPrice() - subTotal)